### PR TITLE
TXDensityMaps modifications

### PR DESCRIPTION
**Pull request by tmcornish**
1. Changed how delta_g is calculated.
2. Added option to exclude from the delta_g calculation any pixels whose values are below a user-specified threshold ("weight_thresh" in the config) in the mask .

### CODE REVIEW

File: txpipe/maps.py [hunk 1]
```diff
@@ -436,7 +439,9 @@ def run(self):
         mask[mask == healpy.UNSEEN] = 0
         mask[np.isnan(mask)] = 0
         mask = mask.flatten()
-        pix = np.where(mask > 0)[0]
+        #identify pixels to keep (i.e. mask value above specified threshold)
+        pix_keep = mask > self.config['weight_thresh']
+        pix = np.where(pix_keep)[0]
```
Comment (damonge) [inline txpipe/maps.py:444]: do we use `pix` for anything?

**Review by damonge — COMMENTED** [review]
Just one comment from me!

File: txpipe/maps.py (hunk 1 above)
Comment (tmcornish) [reply txpipe/maps.py:444]: It gets included in the output file but isn't used for any of the calculations in TXDensityMaps.

**Comment by joezuntz** [comment]
This looks great - ignore the test fail, that's a JAX thing, not your problem.

Do you want to convert it from draft to a full PR?

**Review by joezuntz — APPROVED** [review]

**Pull request merged**